docs/source/quickstart.rst

```diff
@@ -28,7 +28,38 @@ For connecting to your `GoPiGo3`_ robot with a computer or laptop, read the inst
 Program your GoPiGo3
 ***********************
 
-For programming your `GoPiGo3`_ to do anything you want, read the instructions found here: `programming your robot`_.
+For programming your `GoPiGo3`_ to do anything you want, you can follow the instructions found here (`programming your robot`_) or *follow the rest of instructions found in this section*.
+
+To install or update the `GoPiGo3`_ library on your RaspberryPi, open a terminal or the command line and type the following command:
+
+.. code-block:: bash
+
+   # follow any given instructions given through this command
+
+   sudo curl -kL dexterindustries.com/update_gopigo3 | bash
+
+Also, in order to be able to use the :py:meth:`easygopigo3.EasyGoPiGo3.init_distance_sensor` method and the :py:class:`easygopigo3.DistanceSensor` class, the `DI-Sensors`_ package is required.
+You can install it or update it with the following command in the terminal:
+
+.. code-block:: bash
+
+   # follow any given instructions given through this command
+
+   sudo curl -kL dexterindustries.com/update_sensors | bash
+
+********************
+Connect More Sensors
+********************
+
+The `GoPiGo3`_ can also be paired with our in-house sensors.
+There are a number of digital and analog sensor that can be connected to the GoPiGo3. We have further in-depth documentation on the following 4 sensors:
```

Comment:
digital and analog sensors  
_(plural is needed)_

Reply:
Done.